buffer/src/test/java/io/netty/buffer/ByteBufUtilTest.java

```diff
@@ -542,6 +539,15 @@ public void testIsTextWithInvalidIndexAndLength() {
         }
     }
 
+    @Test
+    public void testUtf8BytesIsExact() {
```

Comment:
There should be a test that includes a surrogate pair.

Reply:
@ejona86 Added blow this one :+1: